src/hooks/useLocalStorageReducer.ts

```diff
@@ -0,0 +1,38 @@
+import { useCallback, useState } from 'react';
+
+export function useLocalStorageReducer<TValue, TAction>(
```

Comment:
You have two similar hooks, maybe only one is enough?

Reply:
Only one is used, the other one is just chilling there.